paperspace/cli.py

```diff
@@ -181,6 +189,14 @@ def common_experiment_create_multi_node_options(f):
             "--parameterServerRegistryPassword",
             "parameterServerRegistryPassword",
         ),
+        click.option(
+            "--modelType",
+            "modelType",
+        ),
+        click.option(
+            "--modelPath",
+            "modelPath",
+        ),
```

Comment:
This does not have to be repeated. It's defined in common_experiments_create_options

Reply:
Fixed.